From 1cbad6b4387a2a15762e96d2a74ba3762bd97a33 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Wed, 21 Sep 2016 03:52:36 +0000
Subject: [PATCH 230/347] phy: core: fix compilation warning for
 !CONFIG_GENERIC_PHY

This patch comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

A stub implementation of phy_reset() is defined as a non-inline
function for !CONFIG_GENERIC_PHY. This should be an inline
implementation to avoid warnings like,

"../include/linux/phy/phy.h:208:12: warning: 'phy_reset' defined
but not used [-Wunused-function]"

Fixes: 8b3d9976f401 ("phy: core: add API to support phy reset")
Signed-off-by: Suman Anna <s-anna@ti.com>
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
(cherry picked from commit 88988b964b40ac04062b1a50fbf9468d222df295)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 include/linux/phy/phy.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index d6edfd7..f5b6762 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -224,7 +224,7 @@ static inline int phy_init(struct phy *phy)
 	return -ENOSYS;
 }
 
-static int phy_reset(struct phy *phy)
+static inline int phy_reset(struct phy *phy)
 {
 	if (!phy)
 		return 0;
-- 
1.7.5.4

